src/compute/src/compute_state.rs

```diff
@@ -478,7 +481,7 @@ impl<'a, A: Allocate> ActiveComputeState<'a, A> {
                 }
             }
 
-            new_uppers.push((id, new_frontier));
+            new_uppers.push((id, new_frontier.clone()));
```

Comment:
Now we avoid the allocation above but need to `.clone()` here. How is this different in terms of memory footprint?

Reply:
Most of the iterations don't even get here because we only report changes.